software/wmla120.py

```diff
@@ -241,6 +205,40 @@ def __init__(self, eval_ver=False, non_int=False, arch='ppc64le', proc_family=''
 
         self.log.debug(f'software variables: {self.sw_vars}')
 
+    def get_state(self):
```

Comment:
Name should probably start with an '_'  ie _get_state.  A better name might be _get_state_dict

Reply:
done